qualtran/bloqs/chemistry/sparse/prepare.py

```diff
@@ -77,39 +80,84 @@ class PrepareSparse(Bloq):
     num_bits_state_prep: int
     num_bits_rot_aa: int = 8
     adjoint: bool = False
-    k: int = 1
+    qroam_block_size: Optional[int] = None
 
     @cached_property
-    def signature(self) -> Signature:
-        return Signature(
-            [
-                Register("pqrs", (self.num_spin_orb // 2 - 1).bit_length(), shape=(4,)),
-                Register("theta", 1),
-                Register("alpha", 1),
-                Register("beta", 1),
-                Register("swap_pq", 1),
-                Register("swap_rs", 1),
-                Register("swap_pqrs", 1),
-                Register("flag_1b", 1),
-            ]
+    def selection_registers(self) -> Tuple[SelectionRegister, ...]:
```

Comment:
`Tuple[Register, ...]` is being returned here. Is this ok? If yes: update base class type annotation

Reply:
Fixed.